### ISSUE TITLE
[Regression] Def Jam: Icon Broken Clothing

### ISSUE THREAD

**Issue by Triticum0** [RPCS3/rpcs3#14948]
### Quick summary

The clothing of characters lags behind the player

### Details

When testing this game I ran across an issue where the clothing separated from the character every few seconds

Masters

https://github.com/RPCS3/rpcs3/assets/71127352/fa967a4d-533d-4b37-84d1-a95dea69a3ab

Working build (13535)
![NPEB90001_screenshot_2023_12_27_23_56_48](https://github.com/RPCS3/rpcs3/assets/71127352/29e493ad-aff4-4192-8770-e64901df0083)

### Build with regression

v0.0.22-13536-5c1f79ab

### Attach two log files

[Working.log.gz](https://github.com/RPCS3/rpcs3/files/13782129/Working.log.gz)
[Broken.log.gz](https://github.com/RPCS3/rpcs3/files/13782188/Broken.log.gz)

### System configuration

RPCS3 RPCS3 v0.0.29-15845-475191cf Alpha | master | FW 4.88 | Windows 10
AMD Ryzen 7 5700X 8-Core Processor | 16 Threads | 15.55 GiB RAM | AVX+ | FMA3
GPU: NVIDIA GeForce RTX 3060 Ti (535.104)

### Other details

Changing PPU to interpreter fixes the issue.

**Comment by AniLeo** [comment]
Mentioning PR that introduced the regression: #11904

**Comment by vans936** [comment]
Hi i confirm the problem i installed rpcs3 to play Def Jam Icon but like the author said there are a annoying graphics bug where the player cloths get messy. I respected all the recommanded configuration like turn on the write color buffers. But still the graphic bug reamain. So like the author said there are no problem with the **rpcs3-v0.0.22-13535-799c4837_win64** version but past this version until the recent one there are this texture bug.  Some exemple below

**Def Jam Icon graphic bug issue (video compresed)**
https://github.com/RPCS3/rpcs3/assets/64385560/d43041e1-e822-4fb5-8d03-11bd3bd8685a

**Screenshot**
![dej jam icon issue1](https://github.com/RPCS3/rpcs3/assets/64385560/464d3c61-57ad-4b56-875f-ef91e8e85e62)
![dej jam icon issue2](https://github.com/RPCS3/rpcs3/assets/64385560/6f5db290-b142-42c2-badd-48f0465575a1)
![dej jam icon issue3](https://github.com/RPCS3/rpcs3/assets/64385560/9114056f-88a2-411a-b65f-79b7411c27a6)


**System configuration**

RPCS3 0.0.30-15994 (the latest version) with the last ps3 firmware 4.90/ Windows 11 pro/  gpu AMD 6800xt /cpu amd 7 5800X 3D/ motherboard ROG STRIX B550-I GAMING.

**Build with regression**
since the **v0.0.22-13536-5c1f79ab**

Also i wanted to play with in my Steam Deck so i download the latest linux version but the graphic bug remain so i installed the working version **rpcs3-v0.0.22-13535 linux version** but when it come to select our fighter the game freeze so it doesnt work with that linux version in steam deck.

I dont know how to share or where are located the log file but i might eddit this post.

**Comment by AniLeo** [comment]
Please retest

**Comment by Ctrlsway** [comment]
I also need a fix, besides how slow it runs my only issue is the clothes lagging behind

**Comment by digant73** [comment]
@Triticum0, @Ctrlsway probably still an issue, but please provide an update

**Comment by ghost** [comment]
Any updates on this?

**Comment by vans936** [comment]
2 years laters i play it today to try... the problem is not resolved we still have the broken clothing issue. Also apparently if we try loading a save from the story mode (create a label) the game crash but i dont know about that. As when i see the broken cloth issue i immediately quit after one fight

<img width="2560" height="1440" alt="Image" src="https://github.com/user-attachments/assets/1453bb74-460e-4a5f-9d4b-f54e071db0f6" />
<img width="2560" height="1440" alt="Image" src="https://github.com/user-attachments/assets/06e310ed-c381-4a8c-9bde-c4ad83893b12" />
<img width="2560" height="1440" alt="Image" src="https://github.com/user-attachments/assets/7ed875d5-c056-4e18-8178-f56471a26d49" />

**Comment by vans936** [comment]
YEEEEEEEEEEEESSSSSSSSSS!!!! now it work its resolved! to avoid the graphic glitch the broken clotch etc..  in the gpu tab we need to tick **Write Color Buffers; Strict Rendering Mode and VSync.** Then on the Advanced tab (firmware Libraries) we neet to tick **libvdec.sprx** and in the advanced GPU we need to also tick **Disable Vertex Cache**. tested in windows 11 and linux (steam deck) the game work great. Also aparently the story mode would crash  if we tried to load a save well after few search the solution is to disable auto save and save manualy anyway problem resolved  for me =)

ps: sorry for my bad english peace

**Comment by cammysorammy** [comment]
I'm pretty sure that only Disable Vertex Cache is what fixes the clothes lagging. I've added it to the wiki as a fix, as well as a workaround for the Build a Label crash.

**Comment by vans936** [comment]
> I'm pretty sure that only Disable Vertex Cache is what fixes the clothes lagging. I've added it to the wiki as a fix, as well as a workaround for the Build a Label crash.

Yes you right i just tried your solution only ticked **Disable Vertex cache** is needed to fix the broken clothe problem! also my solution worked too but it added a notifiable input lag (not implayable) but like you said if we only tick **disable vertex cache** the game is smooth with no imput lag Thanks its perfect. As i go to the wiki page https://wiki.rpcs3.net/index.php?title=Def_Jam:_Icon  i dont see your fix if i have time i might add it later to the wiki page
